python/packages/azure_ai_search/agent_framework_azure_ai_search/_search_provider.py

```diff
@@ -111,6 +101,18 @@
 else:
     from typing_extensions import override  # type: ignore[import] # pragma: no cover
 
+"""Azure AI Search Context Provider for Agent Framework.
```

Comment:
Wonder if we can have a ruff rule to make sure our doc strings are below imports. I see it several different ways.